### PR TITLE
move tree to dropdown WIP

### PR DESCRIPTION
**Pull request by fanixus96**
## What did you implement?
Moving whole tree to dropdown 
## How it might be Tested?
1. Create content-type with correct relation (one-way content-type->the-same-content-type)
2. Click on arrow on root element 
3. Check if collapse working fine

## For reviewers

**Files changed & to be reviewed**
->>>> to edit after final commit

### CODE REVIEW

File: extensions/content-manager/admin/src/components/CustomTable/Row.js [hunk 1]
```diff
@@ -71,7 +72,7 @@ const getDisplayedValue = (type, value, name) => {
   }
 };
 
-function Row({ goTo, isBulkable, row, headers, level, relationColumnName }) {
+function Row({ goTo, isBulkable, row, headers, level, relationColumnName, isWrappedWithRoot, manageCollapsedIds, isCollapsed }) {
```
Comment (mstrag) [inline extensions/content-manager/admin/src/components/CustomTable/Row.js]: I will wrap it to next line fro better readability

File: extensions/content-manager/admin/src/components/CustomTable/index.js [hunk 2]
```diff
@@ -36,6 +38,45 @@ const CustomTable = ({ data, headers, isBulkable, rowsToHighlight, relationColum
     tableEmptyMsgId = 'withSearch';
   }
 
+  const isCollapsed = (id) => collapsedIds.find(el => el === id);
+
+  const manageCollapsedIds = (id) => {
+    const isIdCollapsed = isCollapsed(id);
+    if(isIdCollapsed) {
```
Comment (mstrag) [inline extensions/content-manager/admin/src/components/CustomTable/index.js]: ```if(isCollapsed(id))```

**Review by cyp3rius — APPROVED** [review]

File: extensions/content-manager/admin/src/components/CustomTable/Row.js (hunk 1 above)
Comment (fanixus96) [reply extensions/content-manager/admin/src/components/CustomTable/Row.js]: fixed

File: extensions/content-manager/admin/src/components/CustomTable/index.js (hunk 2 above)
Comment (fanixus96) [reply extensions/content-manager/admin/src/components/CustomTable/index.js]: fixed

File: extensions/content-manager/admin/src/components/CustomTable/Row.js [hunk 3]
```diff
@@ -84,6 +85,38 @@ function Row({ goTo, isBulkable, row, headers, level, relationColumnName }) {
 
   const { emitEvent } = useGlobalContext();
 
+  const generateIcoContainerIcons  = () => {
+    let baseIcons = [
+      {
+        icoType: 'pencil-alt',
+        onClick: () => {
+          emitEvent('willEditEntryFromList');
```
Comment (mstrag) [inline extensions/content-manager/admin/src/components/CustomTable/Row.js]: I would rather use const instead of string, like actions in redux,

Comment (fanixus96) [reply extensions/content-manager/admin/src/components/CustomTable/Row.js]: Strapi code

**Review by mstrag — APPROVED** [review]

**Pull request merged**